app/i18n/ja/admin.php

```diff
@@ -187,33 +187,33 @@
 			),
 			'title' => 'ユーザー登録',
 		),
-		'sensitive-parameter' => 'Sensitive parameter. Edit manually in <kbd>./data/config.php</kbd>',	// TODO
+		'sensitive-parameter' => 'センシティブなパラメーターです。<kbd>./data/config.php</kbd> を手動で編集',
 		'tos' => array(
-			'disabled' => 'is not given',	// TODO
-			'enabled' => '<a href="./?a=tos">is enabled</a>',	// TODO
-			'help' => 'How to <a href="https://freshrss.github.io/FreshRSS/en/admins/12_User_management.html#enable-terms-of-service-tos" target="_blank">enable the Terms of Service</a>',	// TODO
+			'disabled' => 'disabled',	// IGNORE
+			'enabled' => '<a href="./?a=tos">enabled</a>',	// IGNORE
```

Comment:
@zukizukizuki Is it on purpose to use English words here?

Reply:
@math-GH 
It can be understood as it is in English, but after thinking about it, I thought it would be more helpful to write the following, so I will modify it.
```
disabled → 無効化
enabled → 有効化
```